AFD-Illinois/ipole
# ipole 2021.3

Opening a PR to indicate intent and keep an eye on automated testing.

ipole has a year's worth of improvements, and we should lock them in.  Notably:
1. Adaptive resolution
2. Fitting functions straight from Symphony, more options
3. Continuous integration/testing
4. Better error handling/messages

There are a couple of things to do first though:
1. Add the analytic comparison/integrator test 
2. Add a higher-resolution test that pushes tetrad creation a little harder
3. Provide a better interface or docs for different emissivities/rotativities. Add e.g. kappa-jet combination as a demo.

There are some pending improvements to adaptive res that should probably land with this, too:
1. Refine on any/all Stokes parameters, rather than purely intensity
2. If we're going to switch the interface from nxmin/nx to nx/refine_level, we should do it now
